### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by arbaran** [Lauriethefish/Quest-Discord-Presence-Client#6]
If I already use Madmagic's client for Quest rich presence (which now has been updated to work with Quest 2 through a SideQuest command line prompt), is installing this client for your Beat Saber mod still necessary? Would it work for me to simply use the other client, or is the mod proprietary to this particular client of yours? Months ago I tried this client and never got it to work, but the other one has been working fine for me. Thanks.

**Comment by Lauriethefish** [comment]
The mod works on either client - with MadMagic's it'll provide more specific information for Beat Saber.
If you have issues with my client, please contact me on discord `@Lauriethefish#6700` for help. I don't do support on github, it's for actual known to be bugs.

**Comment by Lauriethefish** [comment]
Conversation has been finished on Discord.

TL;DR: There were some issues, which were caused by missing core mods. This was also the cause of this client not working.
